docs/spelling_wordlist.txt

```diff
@@ -175,16 +216,25 @@ pre
 precompiler
 prepending
 prerelease
+programmatically
+proptype
 py
 pypi
 pypirc
 quickinstaller
+quickstart
+Quicklinks
 Rapido
+razzle
 redmine
 Redux
 refactor
 release
 renderers
+repo
+repository
+restapi
```

Comment:
This should be an inline literal or part of `plone.restapi` as an inline literal.

```suggestion
```